media/js/mozorg/contribute/contribute-2015-ga.js

```diff
@@ -116,19 +146,24 @@ $(function () {
         };
 
         e.preventDefault();
-        gaTrack(['_trackEvent', 'Mozillian Stories Interactions', 'social button click', person + ' - ' + link], callback);
+        window.dataLayer.push({
+            event: 'mozillian-stories-interaction',
+            browserAction: 'social button click',
+            location: person + ' - ' + link,
+            eventCallback: callback
+        });
     });
 
     // Track other actions on landing page
+    $('.landing-notready .other-actions a')
+        .attr('data-element-location', 'not ready');
```

Comment:
Should chain this `.attr` call with the `.on` call below to curb selector repetition.